packages/components/src/components/Alert/Alert.tsx

```diff
@@ -162,6 +167,34 @@ export const Alert: FC<AlertProps> = ({
     </View>
   )
 
+  const expandIconProps: IconProps = {
+    fill: contentColor,
+    width: Sizing._16,
+    height: Sizing._16,
+    maxWidth: Sizing._24,
+    name: expanded ? 'ChevronUp' : 'ChevronDown',
+  }
+
+  const expandableIcon = (
+    <View style={iconViewStyle}>
+      <Spacer horizontal />
+      <Icon {...expandIconProps} />
+    </View>
+  )
+
+  /**
+   * When an alert is expandable, the content should have additional padding on
+   * the right to appear within the expandable icon. Since the expandable icon
+   * has a maxWidth, this hidden icon matches the spacing of the icon insteading
+   * instead of adding a <Spacer /> with a calculated value.
+   */
+  const spacerIcon = (
+    <View style={iconViewStyle} aria-hidden>
+      <Spacer horizontal />
+      <Icon {...expandIconProps} fill={backgroundColor} />
```

Comment:
Not sure if it matters performance-wise, but wonder if the fill should be `'transparent'`? Not sure if under the hood that would make it not actually render at all (but hopefully respect size) vs. rendering the same color as below.

Reply:
Updated fill color to `'none'`.